src/app/styles/blocks/buttons.scss

```diff
@@ -0,0 +1,59 @@
+@import '../variables.scss';
+
+.button {
+  &_s {
+    &_l {
+      font-size: 1.4em;
+    }
+    &_m {
+      font-size: 1em;
+    }
+    &_s {
+      font-size: 0.8em;
+    }
+  }
+
+  &_no-hover {
+    &::ng-deep .mat-button-focus-overlay {
+      background-color: transparent;
+    }
+  }
+
+  &_c {
+    &_primary {
+      color: $background-blue;
+    }
+
+    &_icon-primary {
+      color: #fff;
+      background-color: $background-blue;
+    }
+  }
+
+  &_hidden {
```

Comment:
I would add global styles, what do you think?

```
.mobile {
  display: none;

  @media screen and (max-width: $tablet) {
    display: block;
  }
}

.desktop {
  display: block;

  @media screen and (max-width: $tablet) {
    display: none;
  }
}
```

Reply:
we can make it next time, let`s add to board tasks